Sources/SnapshotTesting/UIKit.swift

```diff
@@ -63,8 +62,8 @@
   }
 
   extension UIImage: Snapshot {
-    public var snapshotFormat: Data {
-      return self.diffableData
+    public var snapshotFormat: UIImage {
+      return self
```

Comment:
This ensures that images diff appropriately. The two-tiered protocol of `Diffable` and `Snapshot` is complicated! Would love to figure out a simpler way